docker/compose/docker-compose.yml

```diff
@@ -72,8 +72,7 @@ services:
       ZK_URL: zookeeper:2181
       REST_SERVER_PORT: 10080
       JAVA_OPTS: |
-        -Dcontroller.service.port=9090
-        -Dconfig.controller.metricenableCSVReporter=false
```

Comment:
Why did you remove this?

Reply:
Here's the reason: 
* I didn't find `config.controller.metricenableCSVReporter` config param anywhere. So, I assume this is not used. 
* `controller.service.port` is replaced by `controller.service.rpc.listener.port`.